src/Components/DropDown/utils.tsx

```diff
@@ -38,21 +50,65 @@ export const renderInput = (
 };
 export const renderOption = (props: any, option: any) => {
   const { key, ...optionProps } = props;
+
   return (
     <Box
       key={key}
       component="li"
-      sx={{ "& > img": { mr: 2, flexShrink: 0 } }}
+      sx={{
+        display: "flex",
+        alignItems: "center",
+      }}
       {...optionProps}
     >
       {option.image && (
-        <img
-          src={option.image}
-          alt={option.label}
-          style={{ width: 24, height: 24, borderRadius: "50%" }}
-        />
+        <Box
+          sx={{
+            width: 160,
+            display: "flex",
+            alignItems: "center",
+            pr: 2,
+            "& > img": {
+              flexShrink: 0,
+              paddingRight: "4px",
+            },
+          }}
+        >
+          <img
+            src={option.image}
+            alt={option.label}
+            style={{ width: 24, height: 24, borderRadius: "50%" }}
+          />
+          <Typography
+            type={TypographyTypes.BODY_REGULAR}
+            label={option.value}
+          />
+        </Box>
       )}
-      {option.label} {option.value}
+
+      <Typography type={TypographyTypes.BODY_REGULAR} label={option.label} />
     </Box>
   );
 };
+const styles = (theme: Theme) => ({
+  popper: {
+    maxWidth: "fit-content",
+  },
+});
+
+type OptionType = {
+  title: string;
+};
+
+interface PopperMyProps {}
+
+export const PopperMy: React.FC<PopperMyProps> = (props) => {
+  return (
+    <Popper
+      open={false}
+      {...props}
+      style={styles({} as Theme).popper}
+      placement="bottom-start"
+    />
+  );
+};
```

Comment:
there is a lot of mess here, move each part (type, component, styles ) to its place , you know how this is supposed to look.